src/elements/button/button.svelte

```diff
@@ -46,9 +45,29 @@ const handleClick = () => {
 const handleParentClick = (event: PointerEvent) => {
   event.stopImmediatePropagation();
 };
+
+let fill = '';
+
+switch (variant) {
```

Comment:
We should wrap this in a `$` so that it's reactive (e.g. if a user updates the `variant` prop we'd want fill to update)

```ts
$: {
  switch (variant) { ... }
}